src/maps/deployment.ts

```diff
@@ -8,9 +9,10 @@ import { MapFunction } from '@/services/Mapper'
 import { Schema } from '@/types/schemas'
 
 export const mapDeploymentResponseToDeployment: MapFunction<DeploymentResponse, Deployment> = function(source) {
+  const parameters = readonly(source.parameters)
```

Comment:
I was noticing that the parameters were getting updated somehow. Like `parametersV2` which should be exactly what is returned from the api was being modified by something. By wrapping the parameters in `readonly` that fixed the issue. I was hoping I'd see an error pop up to find out what was modifying them. But no errors. Its strange. But this fixed it and didn't want to waste any more time tracking down what old code was accidentally updating the parameters by an object reference.